pkg/providers/instancetype/instancetype.go

```diff
@@ -88,6 +88,7 @@ func NewProvider(region string, cache *cache.Cache, ec2api ec2iface.EC2API, subn
 }
 
 func (p *Provider) List(ctx context.Context, kc *corev1beta1.KubeletConfiguration, nodeClass *v1beta1.EC2NodeClass) ([]*cloudprovider.InstanceType, error) {
+	p.cache.DeleteExpired()
```

Comment:
How expensive is this call? Maybe consider benchmarking? If we call it on every list operation, that adds a non-trivial amount of time to the List() call

Reply:
Removed as the cache should be dumped every minute now